dnf-behave-tests/common/output.py

```diff
@@ -124,6 +172,30 @@ def then_dnf_stdout_is(context, dnf_version):
         then_stdout_is(context)
 
 
+@behave.then("dnf4 stdout matches line by line")
```

Comment:
I am not sure if you prefer this approach but just adding a note that all the new steps could use the `{dnf_version:dnf_version}` pattern just like `@behave.then("{dnf_version:dnf_version} stdout is")`.

Reply:
didn't know, I definitely like it more. thanks